lib/experimental/sync/README.md

```diff
@@ -0,0 +1,175 @@
+# Signaling Server Documentation
+
+The signaling server allows multiple clients to exchange messages with each other through various communication topics. Clients can subscribe and unsubscribe to these topics.
+To retrieve messages, clients can send GET requests. They can also perform actions such as subscribing to topics, unsubscribing from topics, publishing messages in specific topics, or pinging the server by sending POST requests.
+Every client must have a unique identifier (which can be randomly generated). This identifier should be included as a parameter named "subscriber_id" in every GET or POST request.
+
+## Sending messages to the server
+
+To send a message to the server, the client needs to send a POST request with its `subscriber_id` and the `message` to be sent.
+The message should be a JSON string that includes its type, topics (or topic, depending on the type), and the data to be transmitted for messages of type publish. The type can be one of the following options: `subscribe`, `unsubscribe`, `publish`, `ping`, depending on the desired action the client wants to execute.
+The details of each action are documented below. If the action is executed successfully by the server, the server will respond with `{"result":"ok"}`.
+  
+### Subscribe to topics
+
+To subscribe to a set of topics, a client must send a POST request with the following parameters:
+
+- `subscriber_id`: Subscriber ID of the client.
+- `message`: 
+    - `type`: Should be set as `subscribe`.
+    - `topics`: An array of topics that the client is no longer interested in reading messages from, e.g., ["WordPress"].
+  
+
+#### Sample request
+
+```js
+
+await (
+	await fetch( 'http://localhost:7888/site-wp-dev/wp-admin/admin-ajax.php', {
```

Comment:
Why is this using admin-ajax instead of the REST API?

Reply:
Because the REST API outputs json and our endpoint is an event stream.